SpatialGDK/Documentation/welcome.md

```diff
@@ -9,9 +9,9 @@ The SpatialOS Game Development Kit (GDK) for Unreal is a <a href="" data-track-l
 The GDK offers:<br/>
 
 * **Multiserver support:** leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/concepts/spatialos), the GDK allows you to run multiple game servers in a single game instance so your Unreal-developed games can have more players, more Actors, and better gameplay systems than previously possible.<br/>
-* **An Unreal-native experience:** keeping traditional workflows and networking APIs that Unreal Engine developers are familiar with, the GDK introduces new native-feeling concepts that turn a single-authority engine into a distributed one. This enables the GDK to retain the functionality of the networking features which Unreal offers out of the box, including transform synchronization, character movement controller, client-side prediction.<br/>
+* **An Unreal-native experience:** keeping traditional workflows and networking APIs that Unreal Engine developers are familiar with, the GDK introduces new native-feeling concepts that turn a single-server engine into a distributed one. This enables the GDK to retain the functionality of the networking features which Unreal offers out of the box, including transform synchronization, character movement, and map travel.<br/>
 * **An easy way to get started:** we have made sure it’s easy to get started with the GDK by including a Starter Project which you can use as a tour of SpatialOS and a base for your own game, as well as a guide to porting your current multiplayer Unreal game to run on SpatialOS.
 
```

Comment:
Need to copypasta this to index.md (for reasons which are too dull to go into) and to the GDK repo's readme.

NOTE: I edited   a sentence in the top point yesterday so this will be new to the GDK repo README:
 "the GDK allows you to run multiple game servers in a single game instance so your Unreal-developed games can have more players, more Actors, and better gameplay systems than previously possible."

Reply:
Had it on my TODOs to edit the GDK repo readme.

Reply:
I made sure the 3 docs reflect the same 3 things